spaces/domains.json

```
@@ -246,5 +246,6 @@
   "vote.crisis.network": "crisisdao.eth",
   "vote.cheesecakeswap.com": "cheesecakeswap.eth",
   "signal.curve.fi": "curve.eth",
-  "voting.snowcrash.finance": "snowcrashdao.eth"
+  "voting.snowcrash.finance": "snowcrashdao.eth",
+  "voting.chimpyswap.com": "chimpyswap"
```

Your domain field value should be similar to your space directory `"my.custom.domain": "my-space.eth"`.
```suggestion
  "voting.chimpyswap.com": "chimpyswap.eth"
```